### PR TITLE
craft.lic - fix for enchanting tier 5

### PR DESCRIPTION
**Pull request by BinuDR**
Updated the tier 5 enchant recipe to buy the correct sigils that base-recipes.yaml now uses.

Fixes this issue: https://github.com/rpherbig/dr-scripts/issues/5216

### CODE REVIEW

File: craft.lic [hunk 1]
```diff
@@ -344,7 +344,7 @@ class Craft
     elsif rank <= 300 # Tier 5 - Basic
       # Buy an abolition and induction sigils and bone totem
       ensure_copper_on_hand(2000, @settings)
-      DRCT.order_item(crafting_data['artificing'][@hometown]['stock-room'], 1) unless DRCI.exists?("induction sigil-scroll")
+      DRCT.order_item(crafting_data['artificing'][@hometown]['stock-room'], 3) unless DRCI.exists?("induction sigil-scroll")
```
Comment (Dartellum) [inline craft.lic:347]: Need to change `induction sigil-scroll` to `congruence sigil-scroll`

File: craft.lic [hunk 2]
```diff
@@ -344,7 +344,7 @@ class Craft
     elsif rank <= 300 # Tier 5 - Basic
       # Buy an abolition and induction sigils and bone totem
```
Comment (Dartellum) [inline craft.lic:345]: Need to change `induction sigil-scroll` to `congruence sigil-scroll`

Comment (BinuDR) [reply craft.lic]: Commited, thanks.

File: craft.lic (hunk 1 above)
Comment (Dartellum) [reply craft.lic:347]: Still need to change the part in the unless.

**Comment by Dartellum** [comment]
The unless part needs updated still. :smile:

**Comment by BinuDR** [comment]
Oh!  Now I see what you mean.

**Comment by BinuDR** [comment]
> The unless part needs updated still. 😄

Taken care of.

**Pull request merged**